tokio-net/src/tcp/listener.rs

```diff
@@ -219,7 +221,7 @@ impl TcpListener {
     ///
     /// # Examples
     ///
-    /// ```
+    /// ```rust,no_run
```

Comment:
Is there a reason `no_run` was added?

Reply:
Primarily because this test kept being run and would always fail since `127.0.0.1:8080` is already in use on my computer. I added the `no_run` after noticing the rest of the doc in this module were also `no_run`.

Reply:
ahh, gotcha. might be good to have a (non-doc) comment explaining that, but not a big deal.